src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/ReadClassifier.java

```diff
@@ -17,7 +17,6 @@
     private static final byte MIN_QUALITY = 15; // minimum acceptable quality in a soft-clip window
     private static final int MAX_LOW_QUALITY_SCORES = 3; // maximum # of low quality base calls in soft-clip window
     private static final float MAX_ZISH_SCORE = 6.f; // maximum fragment-length "z" score for a normal fragment
-    private static final float MIN_CRAZY_ZISH_SCORE = 100.f; // "z" score that's probably associated with a mapping error
```

Comment:
Can we rename `MAX_ZISH_SCORE` to something like `MAX_REGULAR_ZISH_SCORE`? (I'm trying to avoid terms like normal since distributions are involved here).

Reply:
Renamed to `MAX_NON_OUTLIER_ZISH_SCORE`